apps/web/src/services/cost-explorer/model.ts

```diff
@@ -0,0 +1,40 @@
+import type { JsonSchema } from '@spaceone/design-system/types/inputs/forms/json-schema-form/type';
+
+export interface PluginInfoModel {
+    plugin_id: string;
+    version: string;
+    options: Record<string, any>;
+    metadata: Record<string, any>;
+    secret_id: string;
+    upgrade_mode: 'AUTO' | 'MANUAL';
+}
+
+export interface SecretFilterModel {
+    state: 'ENABLED' | 'DISABLED';
+    secrets: string[];
+    service_accounts: string[];
+    schemas: string[];
+}
+
+export interface TemplateModel {
+    cost?: {
+        schema: JsonSchema;
+    };
+}
```

Comment:
suggestion: We decided not to use 'Model' postfix except for api's model.
Actually these are just types that we declare for our convenient usage.
In this file, only DataSourceModel is the model type.
Also, please remove 'Info' in the model name.
We decided embedded type's postfix is 'Info'.

Reply:
I can't understand this rule exactly. Does secretFilter need to have an Info suffix added?